doc/devel/coding_guide.rst

```diff
@@ -148,12 +148,25 @@ We do a backport from master to v2.2.x assuming:
 
 The ``TARGET_SHA`` is the hash of the merge commit you would like to
 backport.  This can be read off of the github PR page (in the UI with
-the merge notification) or through the git CLI tools.::
+the merge notification) or through the git CLI tools.
 
-  git fetch matplotlib
-  git checkout v2.2.x
-  git merge --ff-only matplotlib/v2.2.x
+Assuming that you already have a local branch ``v2.2.x`` (if not, then
+``git checkout -b v2.2.x``), and that your remote pointing to
+``https://github.com/matplotlib/matplotlib`` is called ``upstream``::
+
+  git fetch upstream
+  git checkout v2.2.x  # or include -b if you don't already have this.
+  git reset --hard upstream/v2.2.x
   git cherry-pick -m 1 TARGET_SHA
   # resolve conflicts and commit if required
 
-Use your discretion to push directly to upstream or to open a PR.
+The conflicts will have to be searched for by hand, (search on ``>>>>>``)
```

Comment:
Conflicting files are shown in `git status`, so conflicts only need to be searched in those file, not all of them.

Reply:
Ah, OK, didn't know that.  I'll change to suggesting that.